src/huggingface_hub/commands/scan_cache.py

```diff
@@ -80,60 +80,60 @@ def run(self):
                 print(ANSI.gray(message + " Use -vvv to print details."))
 
     def _print_hf_cache_info_as_table(self, hf_cache_info: HFCacheInfo) -> None:
-        if self.verbosity == 0:
-            print(
-                tabulate(
-                    rows=[
-                        [
-                            repo.repo_id,
-                            repo.repo_type,
-                            "{:>12}".format(repo.size_on_disk_str),
-                            repo.nb_files,
-                            repo.last_accessed_str,
-                            repo.last_modified_str,
-                            ", ".join(sorted(repo.refs)),
-                            str(repo.repo_path),
-                        ]
-                        for repo in sorted(hf_cache_info.repos, key=lambda repo: repo.repo_path)
-                    ],
-                    headers=[
-                        "REPO ID",
-                        "REPO TYPE",
-                        "SIZE ON DISK",
-                        "NB FILES",
-                        "LAST_ACCESSED",
-                        "LAST_MODIFIED",
-                        "REFS",
-                        "LOCAL PATH",
-                    ],
-                )
-            )
-        else:
-            print(
-                tabulate(
-                    rows=[
-                        [
-                            repo.repo_id,
-                            repo.repo_type,
-                            revision.commit_hash,
-                            "{:>12}".format(revision.size_on_disk_str),
-                            revision.nb_files,
-                            revision.last_modified_str,
-                            ", ".join(sorted(revision.refs)),
-                            str(revision.snapshot_path),
-                        ]
-                        for repo in sorted(hf_cache_info.repos, key=lambda repo: repo.repo_path)
-                        for revision in sorted(repo.revisions, key=lambda revision: revision.commit_hash)
-                    ],
-                    headers=[
-                        "REPO ID",
-                        "REPO TYPE",
-                        "REVISION",
-                        "SIZE ON DISK",
-                        "NB FILES",
-                        "LAST_MODIFIED",
-                        "REFS",
-                        "LOCAL PATH",
-                    ],
-                )
-            )
+        print(get_table(self.verbosity, hf_cache_info))
+
+
+def get_table(verbosity: int, hf_cache_info: HFCacheInfo) -> str:
```

Comment:
Slight preference to have `cache_info` first and then `verbosity` as a keyword-only argument. 

```suggestion
        print(get_table(hf_cache_info, verbosity=self.verbosity))


def get_table(hf_cache_info: HFCacheInfo, *, verbosity: int = 0) -> str:
```

Means than you should use it like this `table = get_table(hf_cache_info, verbosity=0)` or even `table = get_table(hf_cache_info)` (more readable / easy to use in my opinion).